app/src/main/java/com/pileproject/drive/preferences/BlockPreferencesSchema.java

```diff
@@ -20,15 +20,13 @@
 
 /**
  * Schema for "block_preferences" table
- *
- * @author <a href="mailto:tatsuyaw0c@gmail.com">Tatsuya Iwanari</a>
- * @version 1.0 21-April-2016
  */
 @Table(name = "block_preferences")
 public class BlockPreferencesSchema {
```

Comment:
I don't understand the whole of codes, but each controller has to have methods like `bool isLineWhite()` if you embed the thresholds in the controller classes. If not, it's better to keep thresholds separated from the controllers because currently `CarController` is defined as it handles and returns values instead of boolean.